src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -199,7 +196,7 @@ function WorkspaceNewRoomPage(props) {
                         >
                             <View style={styles.mb5}>
                                 <RoomNameInput
-                                    ref={(el) => (roomNameInputRef.current = el)}
+                                    ref={inputCallbackRef}
```

Comment:
Coming from [this issue](https://github.com/Expensify/App/issues/42824), we have updated the ref to `ref={inputCallbackRef}`. The inputCallbackRef sets the ref to current and then set the state to true`setIsInputInitialized(true)`. The problem is that, on this screen, the ref is called twice, which leads to setIsInputInitialized(true) being called infinitely. more on this [comment](https://github.com/Expensify/App/issues/42824#issuecomment-2149636667)